src/components/Common/Header.js

```diff
@@ -44,7 +37,7 @@ export default function Header() {
   const [showApprovedOnly, setShowApprovedOnly] = useState(false);
 
   const loadNotifcation = () => {
-    getNotification().then(res => {
+    getNotification().then((res) => {
       if (res.status === 200) {
```

Comment:
1.Not checked for res?.data?.status === 1

Reply:
resolved